subsys/bluetooth/host_extensions/hci_types_host_extensions.h

```diff
@@ -39,6 +39,14 @@ struct bt_hci_cp_set_power_control_request_param {
 	uint8_t apr_margin;
 } __packed;
 
+#define BT_HCI_OP_SET_ROLE_PRIORITY	BT_OP(BT_OGF_VS, 0x011B)
+
+struct bt_hci_set_role_priority {
```

Comment:
This can probably be changed, such that the controller API allows setting multiple priorities at the same time. Right now, if the priority sets grow, then there would be a separate command for each priority in the set.